packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -718,6 +719,10 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
       webexSubServiceType: this.getSubServiceType(meeting),
     };
 
+    if (joinFlowVersion) {
```

Comment:
doesn't this need to be added to the preMeeting function and all the other functions in this file? similar to sessionCorrelationId?

Reply:
just preMeeting function, I think.  Good catch.  Added.